### PR TITLE
add NPE and stack overflow crashes

### PR DESCRIPTION
**Pull request by IrinaShkviro**
Added two more option to the `Crash Test` menu: NPE and StackOverflow.

### CODE REVIEW

File: OrbitQt/orbitmainwindow.cpp [hunk 1]
```diff
@@ -745,5 +745,20 @@ void OrbitMainWindow::on_actionUploadDumpsToServer_triggered(bool checked) {
...
+  CHECK(false);
+}
+
+//-----------------------------------------------------------------------------
+void OrbitMainWindow::on_actionNPE_triggered() {
+  int* null_pointer = nullptr;
+  *null_pointer = 0;
+}
+
+void InfiniteRecursion() {
```
Comment (pierricgimmig) [inline OrbitQt/orbitmainwindow.cpp:757]: Could you add the "//------------------" for consistency?

File: OrbitQt/orbitmainwindow.ui [hunk 2]
```diff
@@ -435,6 +437,16 @@
     <string>Check false</string>
    </property>
   </action>
+  <action name="actionNPE">
+   <property name="text">
+    <string>NPE</string>
```
Comment (pierricgimmig) [inline OrbitQt/orbitmainwindow.ui:442]: What does the E stand for in NPE?  Should we display something more verbose like "nullptr dereference"?

Comment (pierricgimmig) [reply OrbitQt/orbitmainwindow.ui:442]: or "nullptr exception"?

Comment (IrinaShkviro) [reply OrbitQt/orbitmainwindow.ui]: I meant Null Pointer Exception: sorry for Java here. Renamed it now.

File: OrbitQt/orbitmainwindow.cpp (hunk 1 above)
Comment (IrinaShkviro) [reply OrbitQt/orbitmainwindow.cpp]: Fixed, thank you

**Pull request merged**